src/plugins/data/public/ui/query_editor/_query_editor.scss

```diff
@@ -199,3 +199,24 @@
     border: none;
   }
 }
+
+.suggest-widget {
+  position: relative;
+
+  &.visible::after {
+    position: absolute;
+    bottom: -30px;
+    left: 0;
+    width: 100%;
+    height: 30px;
+    background-color: $euiColorLightestShade;
+    border: 1px solid $euiColorLightShade;
+    padding: 5px;
```

Comment:
for example should this be
```
padding: $euiSizeS
```

Reply:
Oh shoot, good catch. The border should also be an eui prop. `$euiBorderThin`, as for padding it should be `$euiSizeXS`. And why do we have padding 5px all around and 4px in the top?

And for font use `@include ouiFontSizeXS`

Reply:
Modified the padding, border, and font size to all use variables. As well as the height and the 'bottom' attribute.